saleor/graphql/checkout/mutations/checkout_lines_update.py

```diff
@@ -130,7 +130,9 @@ def clean_input(
         replace,
     ):
         # if the requestor is not app, the quantity is required for all lines
-        if not info.context.app:
+        if (
+            not info.context.app
+        ):  # TODO: how to use get_app without increasing query count
```

Comment:
Can't you use `load_app` here as well?

Reply:
Yes but it triggers the test checking query count. The other tests (on github) are triggered anyway so I probably should use that here too and try to lower the count in next iteration :thinking:

Reply:
Fixed.